packages/web-react/src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,24 @@
+import classNames from 'classnames';
+import React, { Suspense } from 'react';
+import { useStyleProps } from '../../hooks/styleProps';
+import { useClassNamePrefix, useIconPath } from '../../hooks';
+import { IconProps } from '../../types';
+import { toPascalCase } from '../../utils/strings';
+
+export const Icon = ({ name, size, ...restProps }: IconProps): JSX.Element => {
+  const iconComponentName = toPascalCase(name);
+  const iconPath = useIconPath();
+  const iconClass = useClassNamePrefix('Icon');
+  const { styleProps, props: otherProps } = useStyleProps(restProps);
+  const LazyIcon = React.lazy(() => import(`@lmc-eu/spirit-icons/${iconPath}/${iconComponentName}`));
+
+  return (
+    <Suspense fallback={null}>
+      <div {...otherProps} {...styleProps} className={classNames(iconClass, styleProps.className)}>
```

Comment:
I know it is originally my code, but is there a way to output just the `<svg>...</svg>` element and pass the class directly to it? Also, it would be nice to be able to add a title like in twig.

Reply:
This was reworked as we discussed it in our meeting. Props are passed directly to the SVG element now.